doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -37,6 +37,105 @@ It is recommended to follow the “url” link values,
 applicable to retrieve resources. Instead of constructing your own URLs,
 to keep your client decoupled from implementation details.
 
+## JSON entity representation
+
+Many operations use the same JSON entity representation format which is described in the present section. Each
```

Comment:
An entity is represented by JSON objects which contain the following properties

* id .... represented by a JSON string
* type  .... represented by a JSON string
* A collection of properties, one per entity attribute.